addons/hr_homeworking/models/hr_homeworking.py

```diff
@@ -0,0 +1,285 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, api, fields, models
+from datetime import timedelta
+from odoo.addons.resource.models.utils import string_to_datetime
+DAYS = ['monday_location_id', 'tuesday_location_id', 'wednesday_location_id', 'thursday_location_id', 'friday_location_id', 'saturday_location_id', 'sunday_location_id']
+
+class HrEmployeeLocation(models.Model):
+    _name = "hr.employee.location"
+
+    def _default_employee(self):
+        if 'default_employee' in self.env.context:
+            return self.env.context.get('default_employee')
```

Comment:
but if the key is not in the context, how is it possible to set the value as self.env.user.employee_id.id ?

Reply:
On your field just set:
```py
default=lambda self: self.env.user.employee_id
```

and if the context `default_employee_id` exists, it will be have precedence over the default value defined above :-)
You can check [here how the `default_get` method works](https://github.com/odoo/odoo/blob/a65dda25b95fa904879ca8e7b3309f00ed21fbbf/odoo/models.py#L1354-L1377)